crates/uv/src/commands/project/sync.rs

```diff
@@ -86,10 +87,16 @@ pub(crate) async fn sync(
         concurrency,
     );
 
+    // TODO(konsti): Read editables from lockfile.
```

Comment:
@konstin - Slightly weird interaction here, not sure how it should work exactly...

Reply:
I thought something like `editable: bool` for path dists in the lockfile? Haven't really dug into the format yet